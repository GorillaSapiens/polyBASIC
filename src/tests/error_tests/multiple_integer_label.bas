10 a = 12
10 b = 5
