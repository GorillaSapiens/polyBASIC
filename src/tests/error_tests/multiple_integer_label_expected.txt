ERROR: label ❮10❯ at 2:4 already defined 1:4
